api/backend.go

```diff
@@ -89,23 +89,34 @@ type Backend interface {
 	GetTxLookupInfoAndReceiptInCache(Hash common.Hash) (*types.Transaction, common.Hash, uint64, uint64, *types.Receipt)
 }
 
-func GetAPIs(apiBackend Backend) []rpc.API {
+func GetAPIs(apiBackend Backend, ethAPI *EthereumAPI) []rpc.API {
```

Comment:
This function name starts with `Get`, but it sets something insider for `ethAPI`. How about declaring a new EthereumAPI inside and returning it as a second return value?
`func GetAPIs(apiBackend Backend) ([]rpc.API,  *EthereumAPI)`

Reply:
That sounds great. I modified the source code in https://github.com/klaytn/klaytn/pull/1076/commits/6b9302cb2ab04f7c18c797e7c62edf0a7fc6941c